ow_plexil/src/plans/TestPanTilt.plp

```diff
@@ -0,0 +1,66 @@
+// The Notices and Disclaimers for Ocean Worlds Autonomy Testbed for Exploration
+// Research and Simulation can be found in README.md in the root directory of
+// this repository.
+
+// Simple plan to test Pan and Tilt functions
+
+#include "plan-interface.h"
+
+TestPanTilt: Sequence
+{
+
+    log_info("Testing Pan, should succeed on all tests.");
```

Comment:
This is not always true, since you are not starting with a known pose for the antenna.